examples/dune.inc

```diff
@@ -0,0 +1,333 @@
+(subdir collatz.lv.d
```

Comment:
Do we need to check these in?

Reply:
Check in `dune.inc` files? Yes, dune fails when the are missing. They must at least be existing but empty.

Reply:
Understood, thanks!